mappings/net/minecraft/screen/ScreenHandler.mapping

```diff
@@ -95,6 +95,9 @@ CLASS net/minecraft/class_1703 net/minecraft/screen/ScreenHandler
 	METHOD method_37420 updateToClient ()V
 	METHOD method_37421 getRevision ()I
 	METHOD method_37422 nextRevision ()I
+	METHOD method_37449 setPreviousTrackedSlotMutable (ILnet/minecraft/class_1799;)V
```

Comment:
why mutable suffix?

Reply:
Conflicts with existing `setPreviousTrackedSlot`. (This method does not copy the stack, the existing one does.)

Reply:
Hmm, I don't see why they create a different setter and how this really affects anything beyond reducing object allocation. This name should be good.